Detectors/TRD/base/src/TrackletTransformer.cxx

```diff
@@ -75,10 +75,11 @@ float TrackletTransformer::calculateDy(int detector, int slope, const PadPlane*
   return calibratedDy;
 }
 
-float TrackletTransformer::calibrateX(double x) const
+float TrackletTransformer::calibrateX(int detector, double x) const
 {
-  // Hard-coded value will be replaced once t0 calibration is available from CCDB
-  float t0Correction = -0.279;
+  // NOTE: Calibration value is average over all chambers, stored in PHOS hole (chamber 435)
+  //       option to use chamber-wise values will be implemented in the future
+  float t0Correction = mCalT0->getT0(435);
   return x += t0Correction;
```

Comment:
I think the CI might be stuck, the first 6 checks usually run within a minute.
Maybe you could change 'double x' to 'float x' so that we do get rid of the narrowing warning, commit that and push.
This restarts the CI. Two birds, one stone :).

Reply:
It is not stuck, 1st-time committer should be approved to trigger the CI

Reply:
Just to make sure we don't run into precision errors with the tracklets, is float enough for x? Or should I rather change the t0Correction to double?

Reply:
Ok thanks, I changed the type accordingly